lib/src/source_visitor.dart

```diff
@@ -108,12 +108,45 @@ class SourceVisitor extends ThrowingAstVisitor {
 
   visitAdjacentStrings(AdjacentStrings node) {
     builder.startSpan();
-    builder.startRule();
-    visitNodes(node.strings, between: splitOrNewline);
+    var rule = new Rule();
+    builder.startRule(rule);
+
+    var chunk = zeroSplit();
```

Comment:
Aligning past the keyword would be nice, but I don't want to open that can of worms. If we do it there, users will (not wrongly) want it everywhere, and that gets very hard very quickly.

We also don't want to put a newline after the keyword. That will paint us into a very nasty corner when we later try to make semicolons optional. I think the output should be:

```dart
return "some string ..."
    "next line ...";
```